src/shortcuts/letterCase.ts

```diff
@@ -0,0 +1,23 @@
+import Shortcut from '../@types/Shortcut'
+import Icon from '../components/icons/LetterCaseWithPicker'
+import isDocumentEditable from '../util/isDocumentEditable'
+
+/** Toggle the letter case picker. */
+const letterCase: Shortcut = {
+  id: 'letterCase',
+  label: 'LetterCase',
+  description: 'Change the Letter case.',
+  svg: Icon,
+  canExecute: state => isDocumentEditable() && !!state.cursor,
+  multicursor: {
+    enabled: false,
+    error: () => 'Cannot change text color with multiple thoughts.',
+  },
```

Comment:
You'll want to set this to `multicursor: ignore` and then implement each option as a separate shortcut to enable multicursor support.

I considered the possibility of implementing multicursor support inline to the reducer, but that would miss out on the existing cursor handling that happens automatically with `execShortcutWithMulticursor`. Plus, we need each letter case option to be a separate shortcut anyway so that they show up in the Command Palette. Separate shortcuts is a bit verbose, but it provides the necessary functionality.